charts/elasticsearch-umbrella/charts/elasticsearch-statefulset/templates/configmap.yaml

```diff
@@ -3,10 +3,22 @@ apiVersion: v1
 data:
   {{- toYaml .Values.global.elastic_config | nindent 2 }}
   {{- toYaml .Values.elastic_config | nindent 2 }}
+  {{- if semverCompare ">=7.0.0-0" .Values.image.tag }}
```

Comment:
This is a tricky thing. We're using images with naming like Major.Minor.Path-memlock, it's not standard so the standard semverCompare is not compliant with that. With this expression we're detecting images like:

7.0.0
7.17.0
8.0.0
7.1.1-melock

The last -0 is a wildcard to accept "prereleases". Documentation can be found here https://github.com/Masterminds/semver#working-with-prerelease-versions

There's another option that would be using a regex expression like this one (thanks @marcemv90 for the help with this)

``` {{- if regexMatch "(([7-9](\\.[0-9]+){2})|([0-9]{2,}(\\.[0-9]+){2})).*" .Values.image.tag }}```

It would match with the same expected patterns.

What do you think is better? Feedback welcomed.